src/libs/OptionsListUtils.ts

```diff
@@ -1280,6 +1286,17 @@ function getOptions(
     }
 
     const currentUserOption = allPersonalDetailsOptions.find((personalDetailsOption) => personalDetailsOption.login === currentUserLogin);
+    let recentWorkspaceChats: ReportUtils.OptionData[] = [];
+    let selfDMChat: ReportUtils.OptionData | undefined;
+
+    if (shouldSeparateWorkspaceChat) {
+        recentWorkspaceChats = recentReportOptions.filter((option) => option.isPolicyExpenseChat);
```

Comment:
It looks like we missed this condition: `Your primary workspace should be in the top position`

Reply:
What do you mean by "primary workspace"? Is this already implemented somewhere in the app? I'm just not sure how to find it 😄

Reply:
That's the activePolicyID https://github.com/Expensify/App/blob/8749259618f3f9c31eb817281aea1f482ca26202/src/ONYXKEYS.ts#L154

Reply:
Is similar logic already implemented somewhere in the app? I didn’t have much time today but I’ve done a brief research and I wasn’t able to figure out how I can use that ID to achieve this.

Reply:
Yeah, pretty sure we're using it already to put your default workspace as the top row under "Recents".

Reply:
CC: @mountiny

Reply:
https://github.com/Expensify/App/blob/1f36abb88bb3f99cea8fa3acb82dad8fd9e411cf/src/libs/OptionsListUtils.ts#L937-L951

this I think uses it